api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadServiceDatabase.java

```diff
@@ -190,4 +215,43 @@ public VehicleReadServiceDatabase(VehicleReadDao vehicleReadDao) {
             return null;
         }
     }
+
+    protected DvlaVehicle mapDvlaVehicleSqltoJson(DvlaVehicle vehicle) {
+
+        if (vehicle != null) {
```

Comment:
I know this is something that's in other mappers, but it doesn't make sense. The DB query won't return a null, the resultMapper won't return null, the mapper won't receive null. 
(If I'm wrong with the:

> resultMapper won't return null

 statement, then we should convert the result to empty `List<DvlaVehicle>` list, although I'd be surprised that is the case.